packages/discord.js/src/managers/EntitlementManager.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const { Collection } = require('@discordjs/collection');
+const { makeURLSearchParams } = require('@discordjs/rest');
+const { Routes } = require('discord-api-types/v10');
+const CachedManager = require('./CachedManager');
+const { ErrorCodes, DiscordjsTypeError } = require('../errors/index');
+const { Entitlement } = require('../structures/Entitlement');
+const { SKU } = require('../structures/SKU');
+
+/**
+ * Manages API methods for entitlements and stores their cache.
+ * @extends {CachedManager}
+ */
+class EntitlementManager extends CachedManager {
+  constructor(client, iterable) {
+    super(client, Entitlement, iterable);
+  }
+
+  /**
+   * The cache of this manager
+   * @type {Collection<Snowflake, Entitlement>}
+   * @name EntitlementManager#cache
+   */
+
+  /**
+   * Data that resolves to give an Entitlement object. This can be:
+   * * An Entitlement object
+   * * A Snowflake
+   * @typedef {Entitlement|Snowflake} EntitlementResolvable
+   */
+
+  /**
+   * Data that resolves to give a SKU object. This can be:
+   * * A SKU object
+   * * A Snowflake
+   * @typedef {SKU|Snowflake} SKUResolvable
+   */
+
+  /**
+   * Options used to fetch entitlements
+   * @typedef {Object} FetchEntitlementsOptions
+   * @property {number} [limit] The maximum number of entitlements to fetch
+   * @property {GuildResolvable} [guildId] The id of the guild to fetch entitlements for
+   * @property {UserResolvable} [userId] The id of the user to fetch entitlements for
+   * @property {SKUResolvable[]} [skus] The skus to fetch entitlements for
+   * @property {boolean} [excludeEnded] Whether to exclude ended entitlements
+   * @property {Snowflake} [before] Consider only entitlements before this entitlement id
+   * @property {Snowflake} [after] Consider only entitlements after this entitlement id
+   * <warn>If both `before` and `after` are provided, only `before` is respected</warn>
+   */
+
+  /**
+   * Fetches entitlements for this application
+   * @param {FetchEntitlementsOptions} [options={}] Options for fetching the entitlements
+   * @returns {Promise<Collection<Snowflake, Entitlement>>}
+   */
+  async fetch(options = {}) {
+    const query = makeURLSearchParams({
```

Comment:
- https://github.com/discordjs/discord-api-types/pull/833#discussion_r1367797264

Reply:
That comment was made for the List SKUs endpoint. This is for the List Entitlements endpoint